guide/private-key-management/cloud-backup.md

```diff
@@ -33,14 +33,16 @@ https://www.figma.com/community/file/995256542920917246/BDG---Private-key-manage
    layout = "full-width"
 %}
 
-# Automatic cloud backup
+# Cloud backup
 
 By using an automatic cloud backup for the private key the user is NOT required to manually backup anything when setting up a new [bitcoin wallet]({{ '/guide/glossary/wallet/#bitcoin-wallet' | relative_url }}). This is sometimes also called *seedless*, as the user is never exposed to the traditional *seed*, or [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}) that is used to initiate a wallet.
 
-This scheme can be a good fit for wallets meant for storing smaller amounts of bitcoin. This includes wallets for novices who would find manually backing up a recovery phrase during onboarding confusing, possibly leading to unsafe backups. It could also be a starting point for a progressive security model that lets customers upgrade when they start accumulating more funds. Wallets using automatic cloud backups also work well for what experienced users might call *hot* or *spending* wallets where convenience is more important than maximum security. 
+This scheme can be a good fit for wallets meant for storing smaller amounts of bitcoin. This includes wallets for novices who would find manually backing up a recovery phrase during onboarding confusing, possibly leading to unsafe backups. It could also be a starting point for a progressive security model that lets customers upgrade when they start accumulating more funds. Wallets using automatic cloud backups also work well for what experienced users might call *hot* or *spending* wallets where convenience is more important than maximum security.
 
 The automatic cloud backup scheme gives users reasonable security against theft, while providing low risk for self-inflicted losses.
 
+If a wallet supports Lightning, it may already have cloud backup [enabled for the channel state]({{'/guide/onboarding/backing-up-a-wallet/cloud-backup/' | relative_url}}); therefore, it would not take much additional work to store the private key in the cloud as well.
+
 ### How it works
 First the application generates a wallet with a keypair. Then, the key is encrypted and backed up to a location convenient and safe for the user, yet hard for a malicious third-party to gain access to.
```

Comment:
```suggestion
First the wallet application generates the users private keys. Then, the keys are encrypted and backed up to a location convenient and safe for the user, yet hard for a malicious third-party to gain access to.
```

Wondering if we should put these steps in a list. As well as add a third step around if the user needs to use the backup they do xyz.

Reply:
I'm not sold on this. As far as I understand, all of the "private keys" do not need to be backed up because they can all be deterministically generated from the parent private key. Though I do think we have a slight confusing discrepancy when we say "keypair" vs. "key" so I think I'll just make that "private key".

I don't think the list is necessary, it's a simple tiny paragraph and we explain how the users restores a few lines down.